tests/core/usage_statistics/test_payload_builder.py

```diff
@@ -0,0 +1,97 @@
+from unittest import mock
+
+import pytest
+
+from great_expectations import __version__ as gx_version
+from great_expectations.core.usage_statistics.payload_builder import (
+    UsageStatisticsPayloadBuilder,
+)
+from great_expectations.core.usage_statistics.usage_statistics import (
+    UsageStatisticsHandler,
+)
+from great_expectations.util import get_context
+
+
+def test_build_init_payload(
```

Comment:
First two tests in this file and just shifted from the primary usage stats test file (now that the behavior has shifted to the payload builder)